modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/js/ckeditor4/BaseEditor.js

```diff
@@ -77,6 +77,7 @@ const BaseEditor = forwardRef(
 				},
 			});
 
+			// eslint-disable-next-line react-compiler/react-compiler
```

Comment:
I would really really really really like to not commit this to portal, otherwise people will copy this pattern. Is the refactor required for this very difficult?

Reply:
I can add more "really"s if you think that'd help make my case stronger 😂

Reply:
@bryceosterhaus I pushed a solution that is SF-friendly, based on first render ref. The solution still smells bad, but real refactor would involve `config` not being a controlled prop, and that might be very difficult. This also has a tiny bit worse performance, as ref check is done on each render, but it's pretty negligible.